src/PhenX.EntityFrameworkCore.BulkInsert.Oracle/OracleDialectBuilder.cs

```diff
@@ -0,0 +1,136 @@
+using System.Text;
+
+using Microsoft.EntityFrameworkCore;
+
+using PhenX.EntityFrameworkCore.BulkInsert.Dialect;
+using PhenX.EntityFrameworkCore.BulkInsert.Metadata;
+using PhenX.EntityFrameworkCore.BulkInsert.Options;
+
+namespace PhenX.EntityFrameworkCore.BulkInsert.Oracle;
+
+internal class OracleDialectBuilder : SqlDialectBuilder
+{
+    protected override string OpenDelimiter => "\"";
+    protected override string CloseDelimiter => "\"";
+    protected override string ConcatOperator => "||";
+
+    protected override bool SupportsMoveRows => false;
+
+    public override string CreateTableCopySql(string tempTableName, TableMetadata tableInfo, IReadOnlyList<ColumnMetadata> columns)
+    {
+        var q = new StringBuilder();
+        q.Append($"CREATE TABLE {tempTableName} (");
+
+        foreach (var column in columns)
+        {
+            q.Append($"{column.QuotedColumName} {column.StoreDefinition}");
```

Comment:
I think I have made an extension method for this case AppenJoined(",\n", column, (c, sb) => sb.Append("...")); or something like that